scheduler.rst

```diff
@@ -501,6 +501,103 @@ before being further redispatched to its corresponding handler::
         }
     }
 
+Scheduler Events
+----------------
+
+PreRunEvent
+~~~~~~~~~~~
+
+**Event Class**: :class:`Symfony\\Component\\Scheduler\\Event\\PreRunEvent`
+
+``PreRunEvent`` allows to modify the :class:`Symfony\\Component\\Scheduler\\Schedule`
+or cancel message before it's consumed::
+
+    use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+    use Symfony\Component\Scheduler\Event\PreRunEvent;
+
+    public function onMessage(PreRunEvent $event): void
+    {
+        $schedule = $event->getSchedule();
+        $context = $event->getMessageContext();
+        $message = $event->getMessage();
```

Comment:
Right know, we don't explain in this doc:

- What is a `MessageContext`
- What is a `Schedule` (we talk about ScheduleProvider however)

Reply:
Yes, we're missing things ... but let's do that in other PRs. The idea with the Scheduler docs is to move very fast (even if that breaks some things) to compensate a bit the big delay in merging these docs. Thanks.